HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Dashboard_page.Steps.cs

```diff
@@ -55,7 +55,7 @@ public void When_user_clicks_on_home_link()
 
         public void Then_home_page_should_open()
         {
-            string actualUrl = _driver.WaitUntilPageIsChanged(SeleniumConfiguration.BaseUrl);
+            string actualUrl = _driver.WaitUntilPageIsChanged(SeleniumConfiguration.BaseUrl); 
```

Comment:
It really isn't; I don't have access to AppVeyor (and I did not want to wait). 

This allowed me to force a re-run of AppVeyor. The initial run had a timeout (this was build 36); 

As a side note, I am also investigating a way to ensure time-agnostic (and/or environement) tests